ai-data/generative-apis/api-cli/using-embeddings-api.mdx

```diff
@@ -0,0 +1,54 @@
+---
+meta:
+  title: Using Embeddings API
+  description: This page explains how to use the Embeddings API
+content:
+  h1: Using Embeddings API
+  paragraph: This page explains how to use the Embeddings API
+tags: generative-apis ai-data embeddings-api
+dates:
+  validation: 2024-09-03
+  posted: 2024-09-03
+---
+
+Scaleway Generative APIs are designed as drop-in replacement for the OpenAI APIs. If you have clustering or classification tasks already using one of OpenAI's client libraries, you can easily configure it to point to Scaleway Embeddings API, and get your existing applications running with open-weight embedding models hosted at Scaleway.
```

Comment:
Scaleway Generative APIs are designed as **a** drop-in replacement for the OpenAI APIs